amxmodx/srvcmd.cpp

```diff
@@ -244,6 +278,7 @@ void amx_command()
 		print_srvconsole("   version                    - display amxx version info\n");
 		print_srvconsole("   gpl                        - print the license\n");
 		print_srvconsole("   plugins [ criteria ]       - list plugins currently loaded or ones matching given search criteria\n");
+		print_srvconsole("   plugin < plugin_id >       - information about a plugin\n");
```

Comment:
Why < .. >?

Reply:
@In-line what do you think should be here?

Reply:
`plugin [ id ]`